regex-assembly/942120.ra

```diff
@@ -1,6 +1,9 @@
 ##! Please refer to the documentation at
 ##! https://coreruleset.org/docs/development/regex_assembly/.
 
+##! Please refer to the documentation at
```

Comment:
```suggestion
```

Reply:
Man, this was added by the "regex format" command. We need to check the crs-toolchain to not add these lines of they already exist.

Reply:
The formatter does check. You somehow hit an edge case.